src/test/scala/nl.knaw.dans.easy.multideposit/actions/ValidatePreconditionsSpec.scala

```diff
@@ -215,4 +216,16 @@ class ValidatePreconditionsSpec extends TestSupportFixture with BeforeAndAfterEa
       case Failure(ActionException(message, _)) => message should include("multiple users with id 'dp1'")
     }
   }
+
+  "checkBaseRevisionConformsToUUID" should "not fail if the base revision conforms to UUID pattern" in {
+    val base = testInstructions1.copy(depositorUserId = "dp1").toDeposit().baseUUID
+    val row = testInstructions1.copy(depositorUserId = "dp1").toDeposit().row
+    action.checkBaseRevisionConformsToUUID(testInstructions1.copy(depositorUserId = "dp1").toDeposit(), base) shouldBe Success(())
+  }
+
+  it should "fail if the base revision does not conform to UUID pattern" in {
+    val base = testInstructions2.copy(depositorUserId = "dp1").toDeposit().baseUUID
```

Comment:
I now see why you made `testInstructions2` invalid. However, I think it is better to leave `testInstructions2` valid, as it was before. Then just use `.copy` to create an invalid one.